### PR TITLE
WSO-191 - Hero CTA - Editor refresh

### PR DESCRIPTION
**Pull request by SandyVallee**

### CODE REVIEW

File: sites/all/modules/contrib/maxlength/issue-2267849-13-CKEditor-counterflas-reset-to-max.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/js/maxlength.js b/js/maxlength.js
```
Comment (taeram) [inline sites/all/modules/contrib/maxlength/issue-2267849-13-CKEditor-counterflas-reset-to-max.patch]: .patch files should not be added to the repo. Please remove.

File: sites/all/modules/contrib/maxlength/maxlength.module [hunk 2]
```diff
@@ -3,7 +3,8 @@
 /**
  * @file
  * Limit the number of characters in textfields and textareas and shows the amount of characters left.
- */
+
+ /*Sandy - Testing patch (Maxlength not Working) https://www.drupal.org/files/issues/maxlength-bypass_permision_default_role-2415823-36.patch*/
```
Comment (taeram) [inline sites/all/modules/contrib/maxlength/maxlength.module]: Git commit messages are the preferred way to track who has performed which changes for what reasons, so this comment should be removed.

File: sites/all/modules/contrib/maxlength/p-1.rej [hunk 3]
```diff
@@ -0,0 +1,10 @@
+--- maxlength.js
```
Comment (taeram) [inline sites/all/modules/contrib/maxlength/p-1.rej]: Please remove the `p1-.orig` and `p-1.rej` files, as they're leftovers from the patch process.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/js/maxlength.js [hunk 4]
```diff
@@ -4,430 +4,431 @@
...
-          e.editor.on('elementsPathUpdate', function(e) {
-            setTimeout(function(){ml.ckeditorChange(e)}, 100);
-          });
+    /**
+     * Integrate with ckEditor
+     * Detect changes on editors and invoke ml.calculate()
+     */
+    ml.ckeditor = function() {
+        // We only run it once
+        console.log('running ckeditor '); // doesn't run when editor not working.  Does run when working!!!!!!START HERE
```
Comment (taeram) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/js/maxlength.js]: Please remove debug code.

**Comment by taeram** [comment]
It looks like the file `sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/js/maxlength.js` consists of only whitespace changes and an accidentally committed `console.log()` call. 

Please revert all changes to `maxlength.js`, and manually apply the patch to `maxlength.js` (e.g. open `maxlength.js` in one window, and the patch file in another, and manually copy the changes over)

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/js/maxlength.js (hunk 4 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/js/maxlength.js]: removed

**Comment by SandyVallee** [comment]
I have committed a version with it copied and pasted back from master in the command line window.
I tried manually adding all 3 patches listed in this ticket that were suggested by sr dev. First 2 did not make any difference to the issue listed in the ticket, so I did not re-commit.  The third patch core base is a few versions older then what our module version is so I did not implement it.  It also didn't make sense as it was asking to remove a line that did not exist. I asked Chris for advice and this was his recommendation.  I am currently blocked as to a resolution for this ticket and require Sr. Dev assistance. I have resigned this ticket to Jesse. I did however get some headway in WSO-192 which is related.

File: sites/all/modules/contrib/maxlength/maxlength.module [hunk 5]
```diff
@@ -3,7 +3,6 @@
 /**
  * @file
  * Limit the number of characters in textfields and textareas and shows the amount of characters left.
- */
```
Comment (taeram) [inline sites/all/modules/contrib/maxlength/maxlength.module]: This closing comment likely should be put back in.

**Comment by taeram** [comment]
Please remove the test files as they're unrelated to the fix being applied in this PR.

File: sites/all/modules/contrib/maxlength/maxlength.module (hunk 5 above)
Comment (SandyVallee) [inline sites/all/modules/contrib/maxlength/maxlength.module]: done

**Comment by taeram** [comment]
Please revert the renaming of the `modules/search/tests©search_node_tags.info` and `modules/search/tests©search_node_tags.module` files

**Comment by SandyVallee** [comment]
My patches did not work.  This branch is of no use.  This ticket needs to be reassigned to a sr. dev.

**Pull request closed without merging**